README.md

```diff
@@ -1,10 +1,23 @@
-[![image](https://circleci.com/gh/greenbone/gvm-tools/tree/master.svg?style=svg)](https://circleci.com/gh/greenbone/gvm-tools/tree/master)
+<div align="center">
```

Comment:
align="center" is really from the 90ies. You could try
```html
<div style="display:flex; justify-content: center">
```
instead

Reply:
Thanks